requirements.txt

```diff
@@ -6,7 +6,7 @@ requests
 beautifulsoup4
 numpy
 scipy
-git+https://github.com/fabric8-analytics/nvdlib.git#egg=nvdlib
-git+https://github.com/fabric8-analytics/fabric8-analytics-nvd-toolkit.git#egg=toolkit
-git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git#egg=fabric8-analytics-utils
-git+https://github.com/fabric8-analytics/fabric8-analytics-version-comparator.git#egg=f8a_version_comparator
+git+https://github.com/fabric8-analytics/nvdlib.git@89a9e7e#egg=nvdlib
```

Comment:
Why is this needed for packages you have not updated?

Reply:
@prashbnair We are doing to resolve dependency vulnerability from all CRDA component in phase manner, So we will modify other dependent component as well in near sprints, that's why I pinned all dependent component. Also I have noticed that these component are not changed since many months and don't see any active work also. Felt very safe to pin them to current latest. If you feel otherwise let me know.

Reply:
I am not convinced of this approach. I am approving this for now but we need to have a discussion around this.